### ISSUE TITLE
occupancy versus AoO on indicator graphs

### ISSUE THREAD

**Issue by timadriaens** [trias-project/indicators#89]
Hi, just a detail for the occurrence indicator graphs Y-axis legend: Occupancy, in principle, is mostly used for the probability that a site is occupied (and expressed between 0 and 1) cf. site occupancy models. What we look at in TrIAS is Area of Occupancy the way IUCN use it to quantify range size for species. [This](https://link.springer.com/article/10.1007/s10531-018-1555-5) is an interesting paper about the concept.

@damianooldoni we should probably change the legend for the Y axis to "Area of Occupancy (km2)" to avoid confusion.

**Comment by damianooldoni** [comment]
Yes, @timadriaens. I agree. I will solve it after making new cubes and so having new data.

**Comment by qgroom** [comment]
I thought we just look at the occupancy in any one year. Normally, survey intensity is not great enough to get the true 1km area of occupancy after just one year. The IUCN usually aggregates over a number of years, but we can't do that.

**Comment by damianooldoni** [comment]
Noticed I am still using occupancy. "Measured occupancy" is the term we used in abstract and slides for OEMC 2023 (thanks @qgroom). I would use "measured occupancy" in a next update of the indicators output.
